docs/reference/release-notes/7.3.asciidoc

```diff
@@ -276,19 +276,19 @@ Cluster Coordination::
 * Omit non-masters in ClusterFormationFailureHelper {pull}41344[#41344]
 
 Data Frame::
-* [ML][Data Frame] treat bulk index failures as an indexing failure {pull}44351[#44351] (issue: {issue}44101[#44101])
-* [ML][Data Frame] responding with 409 status code when failing _stop {pull}44231[#44231] (issue: {issue}44103[#44103])
-* [ML][Data Frame] adds index validations to _start data frame transform {pull}44191[#44191] (issue: {issue}44104[#44104])
-* [ML] Data frame task failure do not make a 500 response {pull}44058[#44058] (issue: {issue}44011[#44011])
-* [ML-DataFrame] audit message missing for autostop {pull}43984[#43984] (issue: {issue}43977[#43977])
-* [ML-Data Frame] Add data frame transform cluster privileges to HLRC {pull}43879[#43879]
-* [ML][Data Frame] fix progress measurement for continuous transforms {pull}43838[#43838]
-* [ML][Data Frame] improve pivot nested field validations {pull}43548[#43548]
-* [ML][Data Frame] Adjusting error message {pull}43455[#43455]
-* [Ml Data Frame] Size the GET stats search by number of Ids requested {pull}43206[#43206] (issue: {issue}43203[#43203])
-* [ML-DataFrame] rewrite start and stop to answer with acknowledged {pull}42589[#42589] (issue: {issue}42450[#42450])
-* [ML Data Frame] Set DF task state to stopped when stopping   {pull}42516[#42516] (issue: {issue}42441[#42441])
-* [ML-DataFrame] add support for fixed_interval, calendar_interval, remove interval {pull}42427[#42427] (issues: {issue}33727[#33727], {issue}42297[#42297])
+* Treat bulk index failures as an indexing failure {pull}44351[#44351] (issue: {issue}44101[#44101])
+* Responding with 409 status code when failing _stop {pull}44231[#44231] (issue: {issue}44103[#44103])
+* Adds index validations to _start data frame transform {pull}44191[#44191] (issue: {issue}44104[#44104])
+* Data frame task failure do not make a 500 response {pull}44058[#44058] (issue: {issue}44011[#44011])
+* Audit message missing for autostop {pull}43984[#43984] (issue: {issue}43977[#43977])
+* Add data frame transform cluster privileges to HLRC {pull}43879[#43879]
+* Fix progress measurement for continuous transforms {pull}43838[#43838]
+* Improve pivot nested field validations {pull}43548[#43548]
+* Adjusting error message {pull}43455[#43455]
+* Size the GET stats search by number of Ids requested {pull}43206[#43206] (issue: {issue}43203[#43203])
+* Rewrite start and stop to answer with acknowledged {pull}42589[#42589] (issue: {issue}42450[#42450])
+* Set DF task state to stopped when stopping   {pull}42516[#42516] (issue: {issue}42441[#42441])
```

Comment:
Is the "DF" abbreviation acceptable to use?

Reply:
+1, "DF" seems superflous.